nav2_controller/src/nav2_controller.cpp

```diff
@@ -81,32 +90,31 @@ DwbController::on_configure(const rclcpp_lifecycle::State & state)
 
   // Create the action server that we implement with our followPath method
   action_server_ = std::make_unique<ActionServer>(rclcpp_node_, "FollowPath",
-      std::bind(&DwbController::followPath, this));
+      std::bind(&Nav2Controller::followPath, this));
 
   return nav2_util::CallbackReturn::SUCCESS;
 }
 
 nav2_util::CallbackReturn
-DwbController::on_activate(const rclcpp_lifecycle::State & state)
+Nav2Controller::on_activate(const rclcpp_lifecycle::State & state)
 {
   RCLCPP_INFO(get_logger(), "Activating");
 
-  planner_->on_activate(state);
+  local_planner_->activate();
```

Comment:
The costmap should probably be activated before the controller since the controller requires the costmap

Reply:
Changed.